lib/Conversion/SCFToCalyx/SCFToCalyx.cpp

```diff
@@ -1056,6 +1064,24 @@ LogicalResult BuildOpGroups::buildOp(PatternRewriter &rewriter,
       sitofp.getOut().getType().getIntOrFloatBitWidth(), "signedIn");
 }
 
+LogicalResult BuildOpGroups::buildOp(PatternRewriter &rewriter,
+                                     DivFOp divf) const {
+  Location loc = divf.getLoc();
+  IntegerType one = rewriter.getI1Type(), three = rewriter.getIntegerType(3),
+              five = rewriter.getIntegerType(5),
+              width = rewriter.getIntegerType(
+                  divf.getType().getIntOrFloatBitWidth());
+  auto divFOp = getState<ComponentLoweringState>()
+                    .getNewLibraryOpInstance<calyx::DivSqrtOpIEEE754>(
+                        rewriter, loc,
+                        {/*clk=*/one, /*reset=*/one, /*go=*/one,
+                         /*control=*/one, /*sqrtOp=*/one, /*left=*/width,
+                         /*right=*/width, /*roundingMode=*/three, /*out=*/width,
+                         /*exceptionalFlags=*/five, /*done=*/one});
```

Comment:
Not for this PR, but its really annoying that we have to give these `getI1Type()` for the default wires, e.g., `clk`, `reset`, `done`, and `go`. It'd be nice to have an function that automatically does this for all the primitives, so that users only need to specify the types for the additional wires.